src/main/java/com/espro/flink/consul/jobgraph/JobStatus.java

```diff
@@ -0,0 +1,16 @@
+package com.espro.flink.consul.jobgraph;
```

Comment:
Wrong package should be located in com.espro.flink.consul.jobregistry

Reply:
done